crates/bevy_winit/src/lib.rs

```diff
@@ -350,6 +351,35 @@ pub fn winit_runner(mut app: App) {
                         ),
                     }
                 }
+                WindowEvent::DroppedFile(path_buf) => {
+                    let mut dropped_file_events =
+                        app.resources.get_mut::<Events<DroppedFile>>().unwrap();
+                    let winit_windows = app.resources.get_mut::<WinitWindows>().unwrap();
+                    let window_id = winit_windows.get_window_id(winit_window_id).unwrap();
+                    dropped_file_events.send(DroppedFile {
+                        id: window_id,
+                        path_buf,
+                    });
+                }
+                WindowEvent::HoveredFile(path_buf) => {
+                    let mut hovered_file_events =
+                        app.resources.get_mut::<Events<HoveredFile>>().unwrap();
```

Comment:
I think we should probably combine all drag and drop events into a single event. It seems like order of operations matters here (ex: when did the cancel happen).

(not relevant to this pr, but there are a couple of other events here that we should revisit when it comes to combining/ordering, such as CursorLeft/CursorEntered)